content/trial_getting_started/examples/kubernetes/_index.md

```diff
@@ -0,0 +1,99 @@
++++
+title = "Level 7: Multi-Cluster Kubernetes Orchestration"
+description = "Learn how to design your service topology such that it is abstracted from the infrastructure layer and can be deployed on AWS using a single, flexible template."
+weight = 70
+alwaysopen = false
++++
+
+This example demonstrates defining an environment independent Kubernetes workload and deploying it to development and production clusters.  The example runs on Amazon Web Services (AWS) and so requires and account and API credentials.  It available on [github](https://github.com/cloudify-community/eaas-example).
+
+The infrastructure deployment consists of [`ServiceComponent`s](../../../working_with/service_composition) representing:
+
+ * network
+ * database
+ * object store
+ * kubernetes
+
+The implementation of these components is based on the environment type, which is supplied
+as an input to the deployment.  For example, if a development environment is selected, <a href="http://minio.io">Minio</a> is used as the object store and <a href="https://minikube.sigs.k8s.io/docs/">Minikube</a> as the Kubernetes provider.  If a production environment is selected, <a href="https://aws.amazon.com/s3">S3</a> and <a href="https://aws.amazon.com/eks">EKS</a> is used for each, respectively.
+
+## Concepts
+
+### Configuration Indirection
+
+Multi-Kubernetes Cluster orchestration is achieved using a combination of the plugins mentioned above, and the design technique introduced in the [Environment as a Service](../eaas) example.  The technique uses a blueprint node as the source of cluster configuration.  This configuration is populated by a script which takes the kind of environment desired as an input.  The particulars of the configuration are then looked up and stored in runtime properties on the node.  Other nodes then use this configuration via the `get_attribute` intrinsic function.
+
```

Comment:
Can we change the eaas example to use our kubernetes deployments that are supported? This is also what I asked from Customer Success in their monitoring/scaling example, and they complied. 
https://github.com/cloudify-community/blueprint-examples/tree/master/kubernetes

Reply:
We should.  Not related to this though.  I guess Nir or Nati would prioritize.